tests/unit/async_db/conftest.py

```diff
@@ -61,12 +75,15 @@ def query_data() -> List[List[ColType]]:
             922337203685477580,
             -922337203685477580,
             1,
-            1.0387398573,
+            "1.0387398573",
             "some text",
             "2019-07-31",
+            "2020-07-31",
```

Comment:
New dates extended the range below 1970 so let's reflect this in a test (despite it being a unit test) e.g. `1860-01-31`, which was not possible with regular DATE

Reply:
Sure, agree